src/F1_plotting.py

```diff
@@ -54,13 +60,15 @@ def flows(dict_values, user_input, project_data, results_timeseries, bus, interv
     results_timeseries: pd Dataframe
         Timeseries that is to be plotted
     bus: str,
-        sector that is to be plotted, ie. energyVectors of the energy system - not each and every bus.
+        sector that is to be plotted, ie. energyVectors of the energy system -
+        - not each and every bus.
```

Comment:
Were there two functions for plotting flows? If there weren´t, the docstrings might not make sense right now, because they are actually used to plot each bus and NOT the energyVectors. If you are unsure, we can leave it as-is.

Reply:
Ah, probably you are deleting the function anyway in #512. Nevermind.